README.md

```diff
@@ -0,0 +1,98 @@
+# OpenMME
+## Table of Contents
+[Introduction](#introduction)
+
+[Builds and Installation Procedure](#build-and-installation-procedure)
+
+[Update Config](#update-config)
+
+[Known Issues](#known-issues)
+
+[Config Parameters](#config-parameters)
+
+## Introduction:
+OpenMME is a grounds up implementation of the Mobility management Entity EPC S1 front end to the Cell Tower (eNB). Its design is performance optimized for high speed mobility events over the S1-MME interface, while maintaining state coherent high transaction rate interactions over the S6a interface to the HSS and the S11 interface to the Serving Gateway Control (SGWC). The design allows maximum utilization of the transaction rate allowed by the S1-MME Non-Access Stratum (NAS) messages over SCTP, S6a DIAMETER Attribute Value Pairs (AVPs) over TCP and S11 GTPV2C messages over UDP protocols.
+
+## Build and Installation Procedure
+### Update config : Update following config files
+    {install_path}/openmme/src/mme-app/conf/mme.json
+    {install_path}/openmme/src/s1ap/conf/s1ap.json
+    {install_path}/openmme/src/s11/conf/s11.json
+    {install_path}/openmme/src/s6a/conf/*.json
+    update the Diameter Configuration File:
+        {install_path}/openmme/src/s6a/conf/s6a_fd.conf
+       
+- the following values shall be defined:
+   - Identity:  
+      - the diameter identity.
+      - default value: mme.localdomain
+   - Realm:
+          - the diameter realm (everything past the first period of Identity)
+          - default value: localdomain
+          
+  - Refer free diameter  [config link](http://www.freediameter.net/trac/wiki/Configuration) 
+
+### Build open mme components as follows:
+    cd {install_path}/openmme
+    make clean; make; make install
+
+### Generate certificates   
+The "Diameter Identity" is a fully qualified domain name that is used to access the the Diameter peer. The "Diameter Host" is everything up to the first period of the "Diameter Identity". The  "Diameter Realm" is everything after the first period of the "Diameter Identity".
+
+Execute the following command, using the "Identity" configured in        {install_path}/openmme/src/s6a/conf/s6a_fd.conf
+
+    cd {install_path}/openmme/target/conf
+    ./make_certs.sh <diameter_host> <diameter_realm>
+    For Example: ./make_certs.sh mme localdomain
+
+   ### Start MME modules
+     cd {install_path}/openmme/target/
+     ./run.sh
+
+## Known Issues
```

Comment:
Open github issues instead?

Reply:
i just ported current README.txt. I am not sure if these issues are still valid. Will figure out if these are still valid issues then will open the issues.

Reply:
Do you want to address this in this PR or follow-on PR?

Reply:
Need different PR. Will make sure README.md is in good format. I will keep action item on me to verify the issues and open defect.

Reply:
Opened 2 issues. Other 2 places i need more study and as soon I am clear I will open those issues.